src/app/components/Disclaimer/index.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { ServiceContextProvider } from '../../contexts/ServiceContext';
+import { ToggleContextProvider } from '../../contexts/ToggleContext';
+import ThemeProvider from '../ThemeProvider';
+import { StoryProps } from '../../models/types/storybook';
+
+import metadata from './metadata.json';
+import DisclaimerComponent from '.';
+
+const Component = ({ service, variant }: StoryProps) => {
+  return (
+    <ToggleContextProvider
```

Comment:
wasn't sure how to mock the serviceConfig so this is taking disclaimer text and link etc from the russian serviceConfig.